src/instance.ts

```diff
@@ -576,7 +660,8 @@ class Instance extends common.ServiceObject {
    *   const apiResponse = data[1];
    * });
    */
-  getMetadata(callback) {
+  //tslint:disable-next-line no-any
+  getMetadata(callback: GetInstanceMetadataCallback): any {
```

Comment:
I think you can use `Metadata` instead of `any` here

Reply:
Done !